java/src/jmri/jmrit/logixng/tools/swing/TreeEditor.java

```diff
@@ -41,9 +41,10 @@
     public enum EnableExecuteEvaluate { EnableExecuteEvaluate, DisableExecuteEvaluate }
 
 
-    private final LogixNGPreferences _prefs = InstanceManager.getDefault(LogixNGPreferences.class);
+    // There should only be one clipboard editor open at any time so this is static
+    private static ClipboardEditor _clipboardEditor = null;
```

Comment:
Is access to this going to need any kind of synchronization?

Reply:
Thanks for asking. I have added a comment that it must only be accessed on the GUI thread.